ansible_collections/arista/avd/roles/eos_validate_state/ANTA-Preview.md

```diff
@@ -222,6 +227,21 @@ skipped_tests:
       - VerifyEnvironmentCooling
 ```
 
+## Custom ANTA catalog
+
+You can now provide custom ANTA catalogs to the AVD `eos_validate_state` role. By default, AVD will search for catalog YAML files in the `custom_anta_catalogs` directory and incorporate these tests into the existing dynamically created catalog from AVD. The custom catalog files must be named as follows:
+
+- `<hostname>.yml` or `<hostname>.yaml`
+- `<group>.yml` or `<group>.yaml`
+
```

Comment:
any order or precedence if redefining a test?

Reply:
No. I updated the doc in c490e182faf21a5c7f05d8c9fc38f181a875f708 to explain better.